tools/c7n_azure/tests/azure_common.py

```diff
@@ -12,115 +12,291 @@
 CUSTOM_SUBSCRIPTION_ID = '00000000-5106-4743-99b0-c129bfa71a47'
-DEFAULT_TENANT_ID = 'ea42f556-5106-4743-99b0-c129bfa71a47'
-# latest VCR recording date that tag tests
-# If tests need to be re-recorded, update to current date
-TEST_DATE = datetime.datetime(2018, 9, 10, 23, 59, 59)
+DEFAULT_USER_OBJECT_ID = '00000000-0000-0000-0000-000000000002'
+DEFAULT_TENANT_ID = '00000000-0000-0000-0000-000000000003'
+DEFAULT_STORAGE_KEY = 'DEC0DEDITtVwMoyAuTz1LioKkC+gB/EpRlQKNIaszQEhVidjWyP1kLW1z+jo'\
+                      '/MGFHKc+t+M20PxoraNCslng9w=='
+
+GRAPH_RESPONSE = {
+    "value": [
+        {
+            "NOTE": "THIS RESPONSE FAKED BY AZURE_COMMON.PY",
+            "odata.type": "Microsoft.DirectoryServices.User",
+            "objectType": "User",
+            "objectId": DEFAULT_USER_OBJECT_ID,
+            "displayName": "John Doe",
+            "mail": "john@doe.com",
+            "refreshTokensValidFromDateTime": "2018-08-22T20:37:43Z",
+            "userPrincipalName": "john@doe.com"
+        }
+    ]
+}
+
+ACTIVITY_LOG_RESPONSE = {
+    "value": [
+        {
+            "caller": "john@doe.com",
+            "id": "/subscriptions/ea42f556-5106-4743-99b0-c129bfa71a47/resourcegroups/"
+                  "TEST_VM/providers/Microsoft.Compute/virtualMachines/cctestvm/events/"
+                  "37bf930a-fbb8-4c8c-9cc7-057cc1805c04/ticks/636923208048336028",
+            "operationName": {
+                "value": "Microsoft.Compute/virtualMachines/write",
+                "localizedValue": "Create or Update Virtual Machine"
+            },
+            "eventTimestamp": "2019-05-01T15:20:04.8336028Z"
+        }
+    ]
+}
 
 
 class AzureVCRBaseTest(VCRTestCase):
 
+    TEST_DATE = None
+
+    FILTERED_HEADERS = ['Authorization',
+                        'Accept-Encoding',
+                        'client-request-id',
+                        'retry-after',
+                        'strict-transport-security',
+                        'server',
+                        'User-Agent',
+                        'accept-language',
+                        'Connection',
+                        'x-ms-client-request-id',
+                        'x-ms-correlation-request-id',
+                        'x-ms-ratelimit-remaining-subscription-reads',
+                        'x-ms-request-id',
+                        'x-ms-routing-request-id',
+                        'x-ms-gateway-service-instanceid',
+                        'x-ms-ratelimit-remaining-tenant-reads',
+                        'x-ms-served-by',
+                        'x-aspnet-version',
+                        'x-content-type-options',
+                        'x-powered-by',
+                        'ocp-aad-diagnostics-server-name',
+                        'ocp-aad-session-key',
+                        'vary',
+                        'pragma',
+                        'transfer-encoding',
+                        'expires']
+
+    def is_playback(self):
```

Comment:
doing this as `recording` boolean  attribute will make it orthogonal to the other provider test infrastructure.